src/main/java/com/taboola/async_profiler/api/facade/AsyncProfilerCommandsFactory.java

```diff
@@ -15,14 +15,7 @@ public String createStartCommand(ProfileRequest profileRequest, String filePath)
 
         StringBuilder stringBuilder = new StringBuilder("start");
 
-        Set<String> eventsWithoutLockAndAlloc = new HashSet<>(profileRequest.getEvents());
```

Comment:
Originally this part of code tried to resolve a single event from multi event case, this can be removed.